pilot/lib/pilot/build_manager.rb

```diff
@@ -40,12 +40,17 @@ def upload(options)
       distribute(options, latest_build)
     end
 
-    def distribute(options, build)
+    def distribute(options, build = nil)
```

Comment:
Would be great to change this method over to have named parameters rather than just setting the default value!